examples/features/unmanaged/1_singleton.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import random
+
+from determined.experimental import core_v2 as det
```

Comment:
I am not a fan of this pattern, as we have spent years training users that `import determined as det` is how we think of the world.

I think, "core api" was not called "determined" because "determined" is a whole family of feature offerings, of which core api was only one.  This looks like core_v2 intends on being the only thing in the "determined" category, judging by the naming.  But I don't think that's really what we plan on shipping, is it?  (that is, we don't plan on axing all of det.pytorch, det.keras, det.transformers, etc).